src/Apps/Order/Components/PartnerOfferTimerItem.tsx

```diff
@@ -0,0 +1,63 @@
+import * as React from "react"
+import { useCountdownTimer } from "Apps/Conversations/hooks/useCountdownTimer"
+import StopwatchIcon from "@artsy/icons/StopwatchIcon"
+import { DateTime } from "luxon"
+import { Box, ProgressBar, Spacer, Text } from "@artsy/palette"
+
+export const PartnerOfferTimerItem: React.FC<{
+  startAt: string
+  endAt: string
+}> = ({ startAt, endAt }) => {
+  const { remainingTime, percentComplete } = useCountdownTimer({
+    startTime: startAt,
+    endTime: endAt,
+  })
+
+  const color = "blue100"
+  const endDateTime = DateTime.fromISO(endAt).toLocal()
+  let hour: number
+  if (endDateTime.hour > 12) {
+    hour = endDateTime.hour - 12
+  } else if (endDateTime.hour === 0) {
+    hour = 12
+  } else {
+    hour = endDateTime.hour
+  }
+
+  const minutes =
+    endDateTime.minute < 10 ? "0" + endDateTime.minute : endDateTime.minute
+  const amPm = endDateTime.hour >= 12 ? "pm" : "am"
+  const time = `${hour}:${minutes}${amPm}`
+
+  const actionDeadline = `${endDateTime.monthShort} ${endDateTime.day}, ${time} ${endDateTime.offsetNameShort}`
```

Comment:
Is there any chance we can pull this stuff from 
- Metaphysics, via a formatted time field 
- One of the various timer hooks that we have in Force?

Reply:
(cc @erikdstock - if you wouldn't mind helping track this down)

Reply:
Thank you for the suggestion, @damassi 
I ended up using `luxon` library to format the datetime in this commit: https://github.com/artsy/force/pull/13314/commits/76d33ab702cdaca3f960fd1a8b215e93c780549a

Reply:
Oh no, whatever library used isn't what i mean 😄 Im mean we've got a lot of hooks in force that do things with time, so we don't need to add even more time logic:
- https://github.com/artsy/force/blob/main/src/Utils/Hooks/useCurrentTime.tsx
- https://github.com/artsy/force/blob/main/src/Utils/Hooks/useEventTiming.tsx
- https://github.com/artsy/force/blob/main/src/Utils/Hooks/useTimer.tsx

And then in MP, we've got fields like:

<img width="430" alt="Screenshot 2023-12-22 at 9 41 07 AM" src="https://github.com/artsy/force/assets/236943/0317688c-d4f5-4b91-a766-d385a00769ef">

So all of the formatting happens on the backend, and we just display it. 

Is there something we can do to use one of the above? There are lots of formatted date fields around MP.